tests/cmds/test_legal_hold.py

```diff
@@ -212,6 +246,21 @@ def active_and_inactive_legal_hold_memberships_response(mocker):
     return [_create_py42_response(mocker, ALL_ACTIVE_AND_INACTIVE_CUSTODIANS_RESPONSE)]
 
 
+@pytest.fixture
+def events_response(mocker):
+    return _create_py42_response(mocker, EVENTS_RESPONSE)
+
+
+@pytest.fixture
+def empty_events_response(mocker):
+    return _create_py42_response(mocker, EMPTY_EVENTS_RESPONSE)
+
+
+@pytest.fixture
+def all_events_response(mocker):
```

Comment:
It is better to have mock generator methods actually be generators and not just lists.  I know it probably works fine, but I have seen bugs show up because the tests were doing this. There are some subtle differences.

Reply:
Let me know if I'm still missing the mark here.

Reply:
This works!
You could add more events to the list so that it yields more than once, that way it is more like what will happen in real life (unless there is often only a single event). Not a big deal though.